src/Sylius/Bundle/UiBundle/Resources/translations/messages.en.yml

```
@@ -846,6 +846,7 @@ sylius:
         textarea: 'Textarea'
         thank_you: 'Thank you!'
         thank_you_for_your_order: 'Thank you for your order'
+        thank_you_for_register: 'Thank you for your registration'
```

```suggestion
        thank_you_for_your_registration: 'Thank you for your registration'
```
A bit more consistency